### PR TITLE
Improvements to test v2

### PR DESCRIPTION
**Pull request by gregorwolf**
- Node.JS cli for v2 generation
- ABAP report to test v2 generation

This PR is related to the repository [sap-malware-scanning-openapi](https://github.com/gregorwolf/sap-malware-scanning-openapi) that should be used to create the ABAP definitions for the SAP Malware Scanning Service in the [generated-sap-malware-scanning-openapi](https://github.com/gregorwolf/generated-sap-malware-scanning-openapi) repository. When I copy [MalwareScanAPI.json](https://github.com/gregorwolf/sap-malware-scanning-openapi/blob/main/src/MalwareScanAPI.json) into https://abap-openapi.github.io/web-openapi-client/ the generation works just fine. But when I run it with the script that I've added in this PR or the report then the generation fails.

### CODE REVIEW

**Comment by larshp** [comment]
set `selectionScreenBlockIndentation` in the abaplint configuration for https://rules.abaplint.org/indentation/

**Comment by larshp** [comment]
https://github.com/abaplint/abaplint/issues/2846 opened

File: test_v2/index.mjs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    result.get().clas_icf_impl.get()
+  );
+  fs.writeFileSync(
+    folder + input.get().clas_client.get() + ".clas.abap",
+    result.get().clas_client.get()
+  );
+  fs.writeFileSync(
+    folder + input.get().intf.get() + ".intf.abap",
+    result.get().intf.get()
+  );
```
Comment (larshp) [inline test_v2/index.mjs:69]: suggest some refactoring here, its the same code as in https://github.com/abap-openapi/abap-openapi/blob/main/test_v2/test.mjs#L23

I'll forget changing the index file if its not used in tests

**Review by larshp — CHANGES_REQUESTED** [review]

**Comment by colbyhemond** [comment]
@gregorwolf I was trying out the ABAP program you've added with the malware spec you linked in the original comment. 

I found that if the description fields are too long then it was causing a short dump in the `cl_sxml_string_reader`. I had also attempted to read the JSON spec with `/ui5/cl_json_parser` which resulted in the same issues.

After I reduced the length of the descriptions, it generated like the web client.

* It seems like the manual work around is to reduce the length of the description texts in the JSON specs. 
* TO fix the root of the issue it seems we need to use a different class to read in the JSON spec. -- @gregorwolf or @larshp are you aware of a different class that could read in the JSON file that won't dump because of a value being to long?

**Comment by colbyhemond** [comment]
@gregorwolf after looking into it further it appears the `SOTR_SERV_TABLE_TO_STRING` FM was causing the 255 character restriction that was causing the long descriptions to short dump. In my local copy I replaced it with the code below and it appears to not cause any issues with the schemas I've been testing with.
```abap
DATA json_string TYPE string.
LOOP AT data_tab ASSIGNING FIELD-SYMBOL(<wa>).
    json_string = json_string && <wa>.
ENDLOOP.
```

**Comment by larshp** [comment]
closing, feel free to open new PR which passes CI

**Pull request closed without merging**